cypress/support/fragments/oai-pmh/oaipmhPane.js

```diff
@@ -1,25 +1,56 @@
-import { NavListItem, Pane } from '../../../../interactors';
+import { NavListItem, Pane, Section } from '../../../../interactors';
+import General from './general';
+import Technical from './technical';
+import Behavior from './behavior';
+import Logs from './logs';
 
+const navPaneSection = Section({ id: 'app-settings-nav-pane' });
 const oaipmhPane = Pane('OAI-PMH');
-const generalListItem = NavListItem('General');
-const technicalListItem = NavListItem('Technical');
-const behaviorListItem = NavListItem('Behavior');
+
+export const SECTIONS = {
+  GENERAL: 'General',
+  TECHNICAL: 'Technical',
+  BEHAVIOR: 'Behavior',
+  LOGS: 'Logs',
+};
+
+const oaiPmhSections = {
+  [SECTIONS.GENERAL]: General,
+  [SECTIONS.TECHNICAL]: Technical,
+  [SECTIONS.BEHAVIOR]: Behavior,
+  [SECTIONS.LOGS]: Logs,
+};
 
 export default {
   waitLoading() {
     cy.expect(oaipmhPane.exists());
   },
+  checkPageTitle(title) {
+    cy.title().should('eq', title);
+  },
+  checkSectionListItems({ canViewLogs = false } = {}) {
+    cy.expect(oaipmhPane.exists());
 
-  verifyPaneElements() {
-    cy.expect([
-      oaipmhPane.exists(),
-      oaipmhPane.find(generalListItem).exists(),
-      oaipmhPane.find(technicalListItem).exists(),
-      oaipmhPane.find(behaviorListItem).exists(),
-    ]);
+    [
+      SECTIONS.GENERAL,
+      SECTIONS.TECHNICAL,
+      SECTIONS.BEHAVIOR,
+      ...(canViewLogs ? [SECTIONS.LOGS] : []),
+    ].forEach((section) => {
+      cy.expect(navPaneSection.find(NavListItem(section)).exists());
+    });
   },
+  selectSection(section) {
+    cy.do(navPaneSection.find(NavListItem(section)).click());
+    cy.expect(Pane(section).exists());
+    // need to wait to prevent application error
+    // eslint-disable-next-line cypress/no-unnecessary-waiting
+    cy.wait(300);
 
-  clickBehaviorItem() {
-    cy.do(behaviorListItem.click());
+    return oaiPmhSections[section];
+  },
+  goToTenantTab() {
```

Comment:
what if to use `TenantPane.goToTenantTab();` instead of creating this method?
they are actually the same

Reply:
This is redundant method, it suppose to be used to open OAI-PMH section from left navigation panel.